apps/gatsby/vite.config.js

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  server: {
+    port: 3000,
+  },
+  base: './',
```

Comment:
Was there a reason the base is written like this instead of an empty string?

Reply:
I like it better this way cause it is more explicit, but really its 6 in one half dozen in the other. I could easily be talked into an empty string if you prefer